### ISSUE TITLE
Created a BOT/Bridge between Mozilla hubs and Matrix

### ISSUE THREAD

**Issue by r3k2** [mozilla/hubs#2611]
This is my second version, 
1st version I was using chromedp but it was super slow and used a lot of resources so I switched to use websockets.
- I been asking many times for help but got none whatsoever from the hubs developers, 
- I am still missing many json documents
- I had to reverse engineer all the json calls with zaproxy and other tools 

But in the end I got a base bridge working for 1 on 1,(for now) I will be adding more features and a two way bridge instead of one way.

I am open to pull requests, actually they will be very appreciated, specially to change all the reversed hacks I have done with the json to sent over websockets since I cant find in the hubs code any of this, and the phx libraries for chat are diff than the mozilla hubs one, seems mozilla hubs have added many layers to this.

https://gitlab.com/rek2/gohubsbot
I am not sure where to post this so people in the hubs community knows about this and maybe help me with it.

Cheers

**Comment by t-900-a** [comment]
Do you have a demo video perhaps? You would get a lot more attention if someone could see this in practice.

Seems really cool! Great work!

**Comment by r3k2** [comment]
@t-900-a yeah I need to improve some things, and I will def make some screenshots, etc. it will help a lot if there is somewhere I can look for the correct json documents instead of having to reverse engineer it by checking the websockets responses and copying.
